Commands.adoc

```diff
@@ -0,0 +1,239 @@
+= Available Commands and Examples
+:toc:
+:toc-placement!:
+
+Once you `vagrant up` and have a running VM, the vagrant-service-manager
+ enables you to check and tweak your container development environment.
+
+'''
+toc::[]
+'''
+
+You can use the vagrant-service-manager to set up your environment variables and
+ get the TLS certificates to secure the Docker communication channel; identify
+ the routable ip network and version of your VM; and check the status and
+ manage the lifecycle of the configured services in the running VM.
+
+To run the vagrant-service-manager plugin use: +
+-------------------------------------------
+vagrant service-manager [command] [options]
+-------------------------------------------
+The following commands can be used: +
+`env`: Displays the connection information for services in the VM +
+`VM`: Displays the VM related information like version, release, IP etc +
```

Comment:
LOL, the command is actually 'box' :-). So there we cannot use VM. Whether 'box' was  good choice or not is a different question. 

Also, can we maybe put this into a table or maybe a labeled list (http://asciidoctor.org/docs/asciidoc-syntax-quick-reference/#lists). It renders really poorly atm.